src/runtime/node/buffer/$cloudflare.ts

```diff
@@ -2,35 +2,21 @@
 import type nodeBuffer from "node:buffer";
 
 export {
-  Blob,
-  File,
   INSPECT_MAX_BYTES,
-  atob,
-  btoa,
```

Comment:
Why do we no longer export these (and others) when they are neither provided by workerd?

Reply:
They are provided by workerd: https://github.com/cloudflare/workerd/blob/main/src/node/buffer.ts

Reply:
I had the same comment... hard to keep up with your simultaneous updates 🤣
Maybe you can add `fixup!` commits instead of fore pushing.

Reply:
![060ec21017e667e3f8041664f8fb7b6b9cc8bbaee97d2e4d120a4871b50911a5](https://github.com/user-attachments/assets/53979b96-49e9-478c-b6ad-3835dada25bc)

Reply:
We squash and also have a nice fellow ([autofix.ci](https://autofix.ci/)) that pushes fixes.